payload/aps_test.go

```diff
@@ -24,13 +24,19 @@ func TestBadgeAndSound(t *testing.T) {
 }
 
 func TestContentAvailable(t *testing.T) {
-	p := payload.APS{
-		ContentAvailable: true,
-	}
+	p := payload.APS{ContentAvailable: true}
 	expected := []byte(`{"aps":{"content-available":1}}`)
 	testPayload(t, p, expected)
 }
 
+func TestCustomArray(t *testing.T) {
```

Comment:
#picky this file looks like a great candidate for [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests)